tests/phpunit/tests/style-engine/styleEngine.php

```diff
@@ -0,0 +1,666 @@
+<?php
+/**
+ * Tests the Style Engine global functions that interact with the WP_Style_Engine class.
+ *
+ * @package WordPress
+ * @subpackage StyleEngine
+ * @since 6.1.0
+ *
+ * @group style-engine
+ */
+
+/**
+ * Tests for registering, storing and generating styles.
+ */
+class Tests_wpStyleEngine extends WP_UnitTestCase {
+	/**
+	 * Tear down after each test.
+	 */
+	public function tear_down() {
+		WP_Style_Engine_CSS_Rules_Store::remove_all_stores();
+		parent::tear_down();
+	}
+
+	/**
+	 * Tests generating block styles and classnames based on various manifestations of the $block_styles argument.
+	 *
+	 * @ticket 56467
+	 * @covers ::wp_style_engine_get_styles
+	 * @covers WP_Style_Engine::parse_block_styles
+	 * @covers WP_Style_Engine::compile_css
```

Comment:
Quick check: While the test may lead to these three being called, is it specifically testing all three? If not, `@covers` should only be used for the specific function(s)/method(s) being tested.

Same question for all instances in the PR.